.github/workflows/ci.yml

```
@@ -13,10 +13,10 @@ jobs:
         uses: actions/checkout@v2
         with:
           submodules: recursive
-      - name: Setup Python 3.8
+      - name: Setup Python 3.11
         uses: actions/setup-python@v2
```

```suggestion
        uses: actions/setup-python@v4
```
^I think this is the current version if we want to update; https://github.com/actions/setup-python